doc-Installing_on_OpenShift_Container_Platform/topics/appe-saml-authentication-example.adoc

```diff
@@ -0,0 +1,105 @@
+[[configmap-settings]]
+=== External Authentication Configuration Map Settings
+
+See <<appe-saml-authentication-example>> for an example configuration map file.
+
+The configuration map includes the following parameters:
+
+* `auth-type` - The authentication type.
```

Comment:
Sorry to nitpick, just a suggestion. I'd format each parameter as "auth-type::" for better readability and to avoid another "-". What do you think?